server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -332,11 +338,17 @@ public void createLogTable(final String tableName, final String entryTypeName)
             StringUtils.format(
                 "CREATE TABLE %1$s (\n"
                 + "  id %2$s NOT NULL,\n"
-                + "  %4$s_id VARCHAR(255) DEFAULT NULL,\n"
-                + "  log_payload %3$s,\n"
+                + "  %3$s_id VARCHAR(255) DEFAULT NULL,\n"
```

Comment:
Will the entry_id ever be null? Maybe it can have NOT NULL in the schema as well